15_perms_lab/permslab.txt

```diff
@@ -1,15 +1,17 @@
 1. Using the file structure in the current directory, write the commands that perform the following:
 
   - change permission of file src/main.c to r--r--r-- using chmod letter notation
-
+chmod a=r src/main.c
   - change permission of file include/player.h to r-xrw--r- using chmod octal (numbers) notation
-
+chmod 564 src/player.c
   - change permission of all files in Doc folder (without changing the permissions of Doc folder itself) to rw-------
-
+chmod 777 Doc/*.*
```

Comment:
No need for the `*.*` - In unix not all files have extension so we use the simpler `Doc/*`

Also you were asked to grant permission rw only, so 777 is the wrong number here. You'll need: `chmod 600 Doc/*`